stringtie.wdl

```diff
@@ -0,0 +1,23 @@
+task Stringtie {
+    String? preCommand
+    File alignedReads
+    File? referenceGFF
+    Int? threads
+
+    command {
+        set -e -o pipefail
+        ${preCommand}
+        stringtie \
+        ${"-p " + threads} \
+        ${"-G " + referenceGFF} \
+        ${alignedReads}
```

Comment:
For better control of the output file maybe ` > ${assembledTranscripts}` is better here. This because the stdout/stderr always endup in the execution directory